app/views/admin/admin_users/show.html.erb

```diff
@@ -22,7 +22,7 @@
 
   <!--main content-->
   <h3 class="heading"><%= ts("Abuse Administration") %></h3>
-  <%= form_tag :action => "update_user", :controller => "admin/admin_users" do %>
+  <%= form_tag :action => "update_status", :controller => "admin/admin_users" do %>
```

Comment:
As long as you're here, could you switch this to the JSON-style syntax like we use in app/views/admin/admin_users/_user_form.html.erb?